daprdocs/content/en/js-sdk-docs/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+type: docs
+title: "JavaScript SDK"
+linkTitle: "JavaScript"
+weight: 1000
+description: JavaScript SDK packages for developing Dapr applications
+no_list: true
+---
+The Dapr JS SDK will allow you to interface with the Dapr process that abstracts several commonly used functionalities such as Service-to-Service invocation, State Management, PubSub, and more.
+
```

Comment:
I would like to see this a bit more detailed. As to explain what the difference is between a server and client. Maybe we can use the following?

# Dapr JS SDK

* **Client:** Create a Javascript client that can interact with a Dapr Sidecar from your application or other Dapr powered applications.
* **Server:** Create a Server that allows a Dapr Sidecar to interact with your application, forwarding event subscriptions, invokes and more.

Thoughts?

Reply:
Good call out. I will add this to the overview page.